scripts/generate_async_api.py

```diff
@@ -48,6 +48,10 @@ def generate(t: Any) -> None:
     print("")
     print(f"    def __init__(self, obj: {class_name}Impl):")
     print("        super().__init__(obj)")
+    print("")
+    print("    def __repr__(self) -> str:")
```

Comment:
You should put it into ImplWrapper once instead of generating it everywhere.

Reply:
This wasn't done by me, @mxschmitt did it I rebased over it

Reply:
@kumaraditya303 yes please add it to the `ImplWrapper` class instead. Thats better! and remove the generation logic which I've implemented.

Reply:
Ok Done ! @mxschmitt